src/camera.rs

```diff
@@ -138,7 +138,7 @@ pub fn trackball_camera(
 				.unwrap_or(trackball.frame);
 			let view = trackball.old_frame.view();
 			transform.translation = view.translation.into();
-			transform.rotation = view.rotation.into();
+			transform.rotation = view.rotation.normalize().into();
```

Comment:
I would instead normalize the `trackball.old_frame` after line [135](https://github.com/qu1x/bevy_trackball/blob/f8264b4ad52c6934848d40af7893265e14d25f19/src/camera.rs#L135) by appending:

```rust
.map(|mut frame| {
	frame.renormalize();
	frame
})
```

In this way, the de-normalization doesn't add-up in the first place as part of the smoothing.

The final snap-in assignment in line 138 shouldn't cause any trouble as `trackball.frame` is already re-normalized in: https://github.com/qu1x/bevy_trackball/blob/f8264b4ad52c6934848d40af7893265e14d25f19/src/constellation.rs#L75